src/events/marketEvents.js

```diff
@@ -7,69 +7,79 @@ const { getTokenName, getTokenSymbol } = require('../services/contractERC20');
+                    getTokenName(web3, event.returnValues.collateralToken), 
+                    getTokenSymbol(web3, event.returnValues.collateralToken), 
+                ])
+                    .then(([conditions, transaction, tokenName, tokenSymbol]) => {
+                        conditions.forEach(condition => {
+                            getQuestion(condition.questionId)
+                                .then((questions) => {
+                                    if (questions.length == 0) {
+                                        console.error(`ERROR: Question for hex "${condition.questionId}" not found on contition ID ${event.returnValues.conditionIds[0]}`);
+                                    } else {
+                                        message.push(questions.map(question => 
+                                            `> *<https://omen.eth.link/#/${condition.fixedProductMarketMakers}|${question.title}>*\n> *Outcomes:*`
+                                        ));
+                                        questions.forEach(question => {
+                                            if (condition.outcomeTokenMarginalPrices) {
+                                                message.push(
+                                                    question.outcomes.map((outcome, i) => 
+                                                        `> \`${(parseFloat(condition.outcomeTokenMarginalPrices[i]) * 100 )
+                                                            .toFixed(2)}%\` - ${outcome}`
+                                                    ));
+                                            } else {
+                                                message.push(
+                                                    question.outcomes.map(outcome => `> - ${outcome}`));
+                                            }
+                                        });
+                                    }
+                                    message.push(`> *Collateral*: <https://${urlExplorer}/token/${event.returnValues.collateralToken}|${tokenName}>`,
+                                        `> *Liquidity*: ${parseFloat(condition.scaledLiquidityParameter).toFixed(2)} ${tokenSymbol}`,
+                                        `> *Created by*: <https://${urlExplorer}/address/${transaction.from}|${truncate(transaction.from, 14)}>`);
+                                        // Send Slack notification
+                                    pushSlackArrayMessages(message);
+                                    console.log(event.returnValues.conditionIds[0] + ':\n' + message.join('\n') + '\n\n');
+                                });
+                        });
                     });
-                });
-        })();
-    });
+            })();
+        });
+    })
+}
+
+/**
+ * Watch `FixedProductMarketMakerCreation` events from a `FixedProductionMarketMakerFactory` contract.
+ */
+module.exports.watchNewMarketsEvent = async (fromBlock) => {
```

Comment:
Maybe you can move this function to some utils file ?

Reply:
https://github.com/gnosis/conditional-tokens-bot/issues/29